prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -19,10 +19,29 @@ postsubmits:
           secretName: compute-image-tools-test-service-account
     max_concurrency: 10
     branches:
-      - ^master$
-
+    - ^master$
 presubmits:
   GoogleCloudPlatform/osconfig:
+  - name: osconfig-presubmit-build
+    cluster: gcp-guest
+    trigger: "(?m)^/osconfigbuild$"
```

Comment:
s/osconfigbuild/packagebuild/

Reply:
do you think packagebuild is a bit  generic command for this job? what if we add a new job build a different package?

Reply:
i dont think we will be building another package from this repo..?

Reply:
as discussed offline, changing the name